docker/install_chrome_dependencies.py

```diff
@@ -5,36 +5,28 @@
 """
 import logging
 import subprocess
+from contextlib import suppress
 
 logger = logging.getLogger(__name__)
 
 
 def find_packages(library_name):
-    stdout = run_command(["apt-file", "search", library_name])
+    stdout, _code = run_command(["apt-file", "search", library_name])
     if not stdout.strip():
         return []
     libs = [line.split(":")[0] for line in stdout.strip().split("\n")]
     return list(set(libs))
 
 
 def run_command(cmd, cwd=None, env=None):
-    result = subprocess.run(cmd, cwd=cwd, env=env, capture_output=True, text=True, check=False)
-    return result.stdout
+    result = subprocess.run(cmd, cwd=cwd, env=env, capture_output=True, text=True, check=True)
+    return result.stdout, result.returncode
 
 
 def ldd(file_path):
-    stdout = run_command(["ldd", file_path])
-    # For simplicity, I'm assuming if we get an error, the code is non-zero.
-    try:
-        result = subprocess.run(
-            ["ldd", file_path], capture_output=True, text=True, check=False,
-        )
-        stdout = result.stdout
-        code = result.returncode
-    except subprocess.CalledProcessError:
-        stdout = ""
-        code = 1
-    return stdout, code
+    with suppress(subprocess.CalledProcessError):
+        return run_command(["ldd", file_path])
+    return "", 1
 
 
 raw_deps = ldd("/opt/chrome/chrome")
```

Comment:
Doesn't this also need to check the return code, or does `ldd` return a nonzero status code in cases that aren't really errors for us here? Just thinking we may not want to ignore *all* `ldd` errors.

Reply:
I believe the intention is to return an empty string such we skip the logic of identifying missing packages. We could just allow it to error out instead though If that would be preferable

Reply:
It [appears](https://stackoverflow.com/questions/1256145/ld-linux-verify-exit-codes) that `ldd` exits with a nonzero status code when there's some kind of issue reading the executable (or if it's not a valid executable at all). Unfortunately that StackOverflow answer is all I can find - the [`ldd` man page](https://www.man7.org/linux/man-pages/man1/ldd.1.html) doesn't say anything about exit status codes. I suppose we can wait until the executable fails to run since that might give us a more descriptive error